src/widgets/video/VideoStreamTile.tsx

```diff
@@ -30,8 +30,8 @@ const definition = defineWidget({
     },
   },
   gridstack: {
-    minWidth: 3,
-    minHeight: 2,
+    minWidth: 1,
```

Comment:
I would set two here. Do you think, that there should be really a case where you can set 1?

Reply:
I'd say, let the user decide what he wants as the minimum.
To begin with even making it 1 by 2 is quite small for a video but you never know.